spec/features/consumer_searches_remote_advice_spec.rb

```diff
@@ -0,0 +1,49 @@
+RSpec.feature 'Consumer searches for phone or online advice' do
+  let(:landing_page)        { LandingPage.new }
+  let(:remote_results_page) { RemoteResultsPage.new }
+
+  let(:online_advice) { create :other_advice_method, name: "Advice online (e.g. by video call / conference / email)" }
+  let(:phone_advice)  { create :other_advice_method, name: "Advice by telephone" }
+
+  scenario 'Selecting online advice' do
+    with_elastic_search! do
+      given_i_am_on_the_rad_landing_page
+      and_firms_providing_remote_services_were_previously_indexed
+      when_i_submit_a_search_selecting_online_advice
+      then_i_am_shown_firms_and_subs_that_provide_advice_online
```

Comment:
Can we change this so just `then_i_am_shown_firms_that_provide_advice_online` -- we know now that subsidiaries are just another type of firm, so we dropped mentioning that around the various places in the application and specs. I know this is part of the story text, so that should be changed also!